frame/nomination-pools/src/weights.rs

```diff
@@ -18,8 +18,8 @@
 //! Autogenerated weights for pallet_nomination_pools
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2022-06-15, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
-//! HOSTNAME: `bm3`, CPU: `Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz`
+//! DATE: 2022-08-31, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! HOSTNAME: `<UNKNOWN>`, CPU: `<UNKNOWN>`
```

Comment:
If I understand correctly CI will run the benchmark and regenerate this file.

Reply:
correct.